### PR TITLE
feat: remove extensions on editor

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

* Remove extension files

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/test/unit/editor.spec.js [hunk 1]
```diff
@@ -304,6 +304,37 @@ describe('Editor', () => {
...
+              plugin,
+              options: {}
+            }
+          ]
+        });
+
+        expect(plugin).toHaveBeenCalled();
+      });
+    });
+
```
Comment (seonim-ryu) [inline apps/editor/test/unit/editor.spec.js]: @TODO `viewer.js`에도 동일한 스펙 추가

File: apps/editor/test/unit/editor.spec.js [hunk 2]
```diff
@@ -304,6 +304,37 @@ describe('Editor', () => {
...
+      it('should invoke plugin functions', () => {
+        const fooPlugin = jasmine.createSpy('fooPlugin');
+        const barPlugin = jasmine.createSpy('barPlugin');
+
+        editor = new Editor({
+          el: container,
+          plugins: [fooPlugin, barPlugin]
+        });
+
+        expect(fooPlugin).toHaveBeenCalled();
```
Comment (seonim-ryu) [inline apps/editor/test/unit/editor.spec.js]: `toHaveBeenCalledWith` 비교로 변경

File: apps/editor/test/unit/editor.spec.js [hunk 3]
```diff
@@ -304,6 +304,37 @@ describe('Editor', () => {
...
+        expect(barPlugin).toHaveBeenCalled();
```
Comment (dongwoo-kim) [inline apps/editor/test/unit/editor.spec.js]: 이왕이면 플러그인 함수가 호출될 때 어떤 파라미터가 넘어오는지도 같이 테스트하면 좋을 것 같네요.

Comment (seonim-ryu) [reply apps/editor/test/unit/editor.spec.js]: @todo `toHaveBeenCalledWith` 비교로 변경

**Pull request merged**